README.md

```diff
@@ -4,6 +4,8 @@
 
 **Pencil2D** is an animation/drawing software for Windows, macOS, Linux, and FreeBSD. It lets you create traditional hand-drawn animation (cartoon) using both bitmap and vector graphics. Pencil2D is free and open source.
 
+Here is a video demonstrating the project: <https://www.youtube.com/watch?v=ma52j9B1kEM&t=17s>
```

Comment:
Any reason this starts 17 seconds in?

Reply:
I know a lot of youtube users don't like to see "intros" on their binge-watch content, however since Pencil2D is not a typical "youtuber" channel but a project channel with informative purposes, I think it needs to at least keep the branding, and that's considering that most of our reach is fueled by word-of-mouth and how others show the capabilities of the program, either by doing artwork or tutorials.